tests/build.gradle

```diff
@@ -34,9 +34,11 @@ tasks.withType(Test) {
 }
 
 dependencies {
-    compile "org.scala-lang:scala-library:${gradle.scala.version}"
-    compile "org.apache.openwhisk:openwhisk-tests:${gradle.openwhisk.version}:tests"
-    compile "org.apache.openwhisk:openwhisk-tests:${gradle.openwhisk.version}:test-sources"
+    implementation "junit:junit:4.11"
```

Comment:
Ah, i see, this is a option, i will check it in go runtime

Reply:
A parent project (openwhisk core) that uses `implementation` instead of `compile` no longer provides dependencies to child projects. It needs to re-add the required dependencies for testing with `implementation` in sub project.

related documents:
- https://docs.gradle.org/current/userguide/upgrading_version_5.html#dependencies_should_no_longer_be_declared_using_the_compile_and_runtime_configurations
- https://docs.gradle.org/current/userguide/declaring_dependencies.html